File: environment/pset6/mario/less/mario.py
from cs50 import get_int
import cs50

while True:
    n= get_int("Height:")

    if n>=1 and n<=8:




        for i in range(n):
            for j in range(n-i-1):
                print(" ", end="")
            for k in range (n-(n-1)+i):
                print("#", end="")
            print()
        break

File: environment/mario/mario.c
#include <cs50.h>
#include <stdio.h>


int main(void)
{
int n;

do

{
 n=get_int("positive number:");
}
while(n<0 || n>23);

 for (int i=0;i<n;i++)
 {
  for (int j=0;j<n-i-1;j++)
{


printf("-");
}
for (int k=0;k<n-(n-1)+i;k++)
{

 printf("#");
}
for (int r=0;r<n-(n-2);r++)
{
 printf(".");
}
for (int f=0;f<n-(n-1)+i;f++)
{
 printf("#");
}
printf("\n");
}



}
